Solutions/MCBSTM32F400/TinyCLR/scatterfile_tinyclr_mdk.xml

```diff
@@ -114,7 +114,7 @@
             <FileMapping Name="*" Options="(VectorTable +FIRST)" />
         </ExecRegion>
         
-        <ExecRegion Name="ER_RAM_RW" Base="+0" Options="ABSOLUTE" Size="">
+        <ExecRegion Name="ER_RAM_RW" Base="+0" Options="ABSOLUTE NOCOMPRESS" Size="">

```

Comment:
New-ish, and yes it is necessary, As stated in the PR comments this gets around a linker bug for the ARM linker. While the bug is fixed in the latest released version not everyone has migrated to that version and this works around the problem. When we get to the v4.5 release we can require the latest linker as a minimum requirement and verify everything functions properly.
Without this any images built using the older linker will fail to debug apps from VS as the restart to reconnect after deploy will fail triggering a hard fault in the processor.